src/ert/analysis/configuration.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
```

Comment:
Is this needed? :thinking:

Reply:
No, had to rebase do to conflicts on main, remove it in that operation 👍

Reply:
Remembered now! Is because of :
```python
    @classmethod
    def global_update_step(
        cls, observations: List[str], parameters: List[str]
    ) -> UpdateConfiguration:
```

Reply:
You can use `from typing_extensions` instead and do 
```
@classmethod 
def global_update_step( cls, observations: List[str], parameters: List[str] ) -> Self:
```
which will also cover cases where this is inherited.